backend/apiviews/author_views.py

```diff
@@ -114,6 +114,19 @@ def get_current_user(self, request):
         else:
             return Response({"authenticated": False}, status=status.HTTP_401_UNAUTHORIZED)
 
+    def update_user(self, request, authorId, *args, **kwargs):
+        # author/update/<path:authorId>
+        user_id = protocol_removed(authorId)
+        if request.user.is_authenticated:
+            username = request.data["username"]
+            github_URL = request.data["github_URL"]
+            password = request.data["password"]
+            author = get_object_or_404(User, fullId=user_id)
+            User.objects.filter(fullId=user_id).update(
+                username=username, githubUrl=github_URL, password=password)
+            return Response({"query": "update_user", "success": True, "message": "User updated"}, status=status.HTTP_201_CREATED)
```

Comment:
for put request, we often return 200 instead of 201 since we are updating the existing item

Reply:
kk